cmd/relay/relay/models/models.go

```diff
@@ -0,0 +1,103 @@
+package models
+
+import (
+	"time"
+
+	"gorm.io/gorm"
+)
+
+type DomainBan struct {
+	gorm.Model
+	Domain string `gorm:"unique"`
+}
```

Comment:
Does gorm automatically add a primary key, or could we potentially make `domain` a pkey?

Reply:
It does when the struct extends `gorm.Model`: that adds `ID`, `CreatedAt`, `UpdatedAt`, `DeletedAt`: https://gorm.io/docs/models.html#gorm-Model

I didn't really update this model (it is simple), but think it should be more explicit like the other models, and easy to do this update.